PayPal/src/main/java/com/braintreepayments/api/paypal/RecurringBillingAmountBreakdown.kt

```diff
@@ -0,0 +1,26 @@
+package com.braintreepayments.api.paypal
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * A recurring billing amount breakdown.
+ *
+ * @property itemTotal - Total amount of the items before any taxes or discounts
+ * @property shipping - Cost of shipping the items
+ * @property handling - Cost associated with handling the items (e.g., packaging or processing)
+ * @property taxTotal - Total tax amount applied to the transaction
+ * @property insurance - Cost of insurance applied to the shipment or items
```

Comment:
What do folks think of adding `total` to the end of `shipping`, `handling`, and `insurance`? Reading it now and realizing it may make it more clear to merchants.